app/src/main/java/com/example/marvelapp/data/service/response/MarvelResponse.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.marvelapp.data.service.response
+
+import com.google.gson.annotations.SerializedName
+
+class MarvelResponse(
+        @SerializedName("results") val result: List<CharacterResponse>
```

Comment:
format the code properly pls

Reply:
![image](https://user-images.githubusercontent.com/73836200/109208549-3185a880-7789-11eb-8b08-e1be94b992b8.png)

Mau the code is already formated. Do you want me to aply something specific?